### PR TITLE
[YLD-108][Support Dual Liquidity Provisioning]

### PR DESCRIPTION
**Pull request by desamtralized**
Summary of Changes in PR #40
#### Modal Background Color Change:
- Changed bg-secondary to bg-neutral-100 in the Modal component.
#### Wallet Modal:
- Updated useWalletConnection hook usage to remove passed parameters.
#### Add Liquidity Modal:
- Added dual-sided liquidity support by introducing RUNE balance handling.
- Included additional hooks and utility functions.
- Adjusted UI components to toggle between single-sided and dual-sided liquidity.
- Modified functions to handle dual-sided liquidity, including balance validation and transaction submission.
#### Hooks and Utilities:
- Introduced new hook useThorchain.
- Updated useWalletConnection to manage local storage for network addresses.
- Enhanced liquidity position management in useLiquidityPosition.

### CODE REVIEW

File: app/app/explore/components/AddLiquidityModal.tsx [hunk 1]
```diff
@@ -55,6 +61,14 @@ export default function AddLiquidityModal({
     return null;
   }, [pool.asset]);
 
+  const assetMinimalUnit = useMemo(() => {
+    return 1 / 10 ** poolNativeDecimal;
+  }, [poolNativeDecimal]);
+
+  const runeMinimalUnit = useMemo(() => {
```
Comment (MattNguyen) [inline app/app/explore/components/AddLiquidityModal.tsx]: don't need to useMemo here right?

File: app/app/explore/components/AddLiquidityModal.tsx [hunk 2]
```diff
@@ -55,6 +61,14 @@ export default function AddLiquidityModal({
     return null;
   }, [pool.asset]);
 
+  const assetMinimalUnit = useMemo(() => {
+    return 1 / 10 ** poolNativeDecimal;
```
Comment (MattNguyen) [inline app/app/explore/components/AddLiquidityModal.tsx]: not fully convinced that we need useMemo for this too, but ok leaving it

File: app/utils/wallet/chainConfig.tsx [hunk 3]
```diff
@@ -60,6 +60,7 @@ const singleWalletChains = [
   {
     id: "dogecoin",
     name: "Dogecoin",
+    thorchainIdentifier: "doge",
```
Comment (MattNguyen) [inline app/utils/wallet/chainConfig.tsx:63]: Why can't we use id here?

File: app/app/explore/components/AddLiquidityModal.tsx (hunk 1 above)
Comment (desamtralized) [reply app/app/explore/components/AddLiquidityModal.tsx]: Yeah, not really.

File: app/app/explore/components/AddLiquidityModal.tsx (hunk 2 above)
Comment (desamtralized) [reply app/app/explore/components/AddLiquidityModal.tsx]: It can be a one time computed const.

File: app/utils/wallet/chainConfig.tsx (hunk 3 above)
Comment (desamtralized) [reply app/utils/wallet/chainConfig.tsx:63]: We need a way to match the short name from thorchain to our configs, "eth, bsc, doge, btc".

**Pull request merged**